dev/how-to/get-started/TiDB-Binlog-tutorial.md

```diff
@@ -0,0 +1,419 @@
+---
```

Comment:
Please rename the `TiDB-Binlog-tutorial.md` to `tidb-binlog.md`. Generally, we'll keep the name in docs-cn with that in docs.